app/src/main/kotlin/com/thundermaps/apilib/android/impl/AndroidClient.kt

```diff
@@ -39,7 +39,7 @@ class AndroidClient @Inject constructor() {
     @io.ktor.util.KtorExperimentalAPI
     fun client(params: RequestParameters): Pair<HttpClient, HttpRequestBuilder> {
         // Reinitialize if users credentials have changed
-        if (currentCredentials != params.credentials) {
+//        if (currentCredentials != params.credentials) {
```

Comment:
Why do we need to comment these code?

Reply:
This is the root cause every time the headers will pass the value 0-99 as range

Reply:
We need to check the header, don't force update every time